virttest/utils_config.py

```diff
@@ -101,7 +102,12 @@ def __init__(self, path):
         self.parser.optionxform = str
         self.backup_content = open(path, 'r').read()
         read_fp = StringIO('[root]\n' + self.backup_content)
-        self.parser.readfp(read_fp)
+        #readfp function has been deprecated since python3.2 
+        #and removed in python3.12
+        if sys.version_info >= (3, 12):
+            self.parser.read_file(read_fp)
+        else:
+            self.parser.readfp(read_fp)
```

Comment:
Hi @Anushree-Mathur thank you for fixing the issue! Just a minor comment regarding the code change

```suggestion
        self.parser.read_file(read_fp)
```

as `read_file` was introduced since python 3.2 and at the moment avocado-vt only support the version > 3.7

Reply:
@luckyh Thank you for the review! It was only deprecated since python3.2! I tried the same code for python versions such as python 3.7 3.8 till 3.12. But this problem only came for Python3.12 version as this readfp has been removed since python3.12. That is why I mentioned this condition in the code itself. Hope i am able to clarify my point!
Could you please let me know if you agree with it!

Reply:
I see. However, the reason I suggested as so is that the else clause will become a dead code after dropping the support of the python versions prior to 3.12 in the future, which would request another PR for the removal, while the suggested approach works for both the cases.

What do you think?